.github/workflows/push-changes.yaml

```diff
@@ -0,0 +1,25 @@
+name: Push Changes
+
+on:
+  push:
+    branches: [main]
+  workflow_run:
+    workflows:
+      - Cypress
+      - Docker
+    types:
+      - completed
+
+jobs:
+  dispatch:
+    strategy:
+      matrix:
+        repo: ['devfile/registry']
```

Comment:
This needs to go to registry-support, as we need to rebuild the the devflie index base image that contains the registry viewer.

Reply:
Fixed